src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -520,18 +524,18 @@ class ContentExplorer extends Component {
             // is selected and that item's isActionDisabled is false, we enable the action button
             areActionButtonsDisabled =
                 (selectedItemsIds.length === 0 && !isNoSelectionAllowed) ||
-                (selectedItemsIds.length === 1 && selectedItems[selectedItemsIds[0]].isActionDisabled);
+                (selectedItemsIds.length === 1 && allSelectedItems[selectedItemsIds[0]].isActionDisabled);
```

Comment:
I was wondering if we could extract `allSelectedItems[selectedItemsIds[0]].isActionDisabled` to a variable since it's used in 3 places below (I know, this was like that before):
```
   const isActionDisabled = allSelectedItems[selectedItemsIds[0]]?.isActionDisabled;
```